backend/backend.go

```diff
@@ -162,10 +165,15 @@ func NewBackend(arguments *arguments.Arguments, environment Environment) (*Backe
 		return nil, err
 	}
 	backend.notifier = notifier
+	backend.socksProxy = socksproxy.NewSocksProxy(backend.config.AppConfig().Backend.UseProxy, backend.config.AppConfig().Backend.ProxyAddress)
+	backend.log.Println("Creating updater instance")
+	_ = NewRatesUpdater(backend.socksProxy)
```

Comment:
If you remove the singleton, then this workaround/hack would not be necessary. Just keep the rates updater in the backend struct.

Could be a separate commit to make the rates not a global var, but a part of the Backend struct.

Reply:
Already started on this, hope to have it finished by tonight.

Reply:
Do you have an opinion on how to pass the rates to the coin/btc handlers for the coin conversion rates?

Reply:
Either pass it to the NewAccount() constructors, or pass to account.Initialize() as an argument.

accounts.Interface then will have a `RatesUpdader() *backend.RatesUpdater` method (or `rates.Updater` if moved to its own package).

Reply:
I went with creating a new rates package and passing the `rateUpdater` struct to the `NewAccount` constructors.